src/pages/cheoljinJu/Login/Login.js

```diff
@@ -1,9 +1,91 @@
 import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import './Login.scss';
+import '../../../styles/reset.scss';
+import '../../../styles/common.scss';
 
 class LoginCheoljin extends Component {
+  state = {
+    id: '',
+    password: '',
+  };
+
+  handleInput = event => {
+    const { className, value } = event.target;
+    const state = { ...this.state, [className]: value };
+    this.setState(state);
```

Comment:
- 클래스형 컴포넌트에서는 setState를 하면서 인자로 넘겨준 객체를 기존의 state안에 `병합`을 합니다. 따라서, 기존의 값들을 넣어주는 `...this.state` 과정을 거치지 않아도 됩니다.
```suggestion
    this.setState({ [className]: value });
```

Reply:
멘토님 말씀처럼 필요없는 과정을 추가했던 것 같습니다. 수정해서 간결하게 작성될 수 있도록 적용하겠습니다! 감사합니다.